Vision/src/V1/SafeSearchAnnotation.php

```diff
@@ -295,197 +229,5 @@ public function setRacy($var)
-    {
-        @trigger_error('medical_confidence is deprecated.', E_USER_DEPRECATED);
-        GPBUtil::checkFloat($var);
-        $this->medical_confidence = $var;
-
-        return $this;
-    }
-
-    /**
-     * Confidence of violence_score. Range [0, 1]. 0 means not confident, 1 means
-     * very confident.
-     *
-     * Generated from protobuf field <code>float violence_confidence = 22 [deprecated = true];</code>
-     * @return float
-     * @deprecated
-     */
-    public function getViolenceConfidence()
-    {
-        @trigger_error('violence_confidence is deprecated.', E_USER_DEPRECATED);
-        return $this->violence_confidence;
-    }
-
-    /**
-     * Confidence of violence_score. Range [0, 1]. 0 means not confident, 1 means
-     * very confident.
-     *
-     * Generated from protobuf field <code>float violence_confidence = 22 [deprecated = true];</code>
-     * @param float $var
-     * @return $this
-     * @deprecated
-     */
-    public function setViolenceConfidence($var)
-    {
-        @trigger_error('violence_confidence is deprecated.', E_USER_DEPRECATED);
-        GPBUtil::checkFloat($var);
-        $this->violence_confidence = $var;
-
-        return $this;
-    }
-
-    /**
-     * Confidence of racy_score. Range [0, 1]. 0 means not confident, 1 means very
-     * confident.
-     *
-     * Generated from protobuf field <code>float racy_confidence = 24 [deprecated = true];</code>
-     * @return float
-     * @deprecated
-     */
-    public function getRacyConfidence()
-    {
-        @trigger_error('racy_confidence is deprecated.', E_USER_DEPRECATED);
-        return $this->racy_confidence;
-    }
-
-    /**
-     * Confidence of racy_score. Range [0, 1]. 0 means not confident, 1 means very
-     * confident.
-     *
-     * Generated from protobuf field <code>float racy_confidence = 24 [deprecated = true];</code>
-     * @param float $var
-     * @return $this
-     * @deprecated
-     */
-    public function setRacyConfidence($var)
-    {
-        @trigger_error('racy_confidence is deprecated.', E_USER_DEPRECATED);
-        GPBUtil::checkFloat($var);
-        $this->racy_confidence = $var;
-
-        return $this;
-    }
-
-    /**
-     * Confidence of nsfw_score. Range [0, 1]. 0 means not confident, 1 means very
-     * confident.
-     *
-     * Generated from protobuf field <code>float nsfw_confidence = 26 [deprecated = true];</code>
-     * @return float
-     * @deprecated
-     */
-    public function getNsfwConfidence()
-    {
-        @trigger_error('nsfw_confidence is deprecated.', E_USER_DEPRECATED);
-        return $this->nsfw_confidence;
-    }
-
-    /**
-     * Confidence of nsfw_score. Range [0, 1]. 0 means not confident, 1 means very
-     * confident.
-     *
-     * Generated from protobuf field <code>float nsfw_confidence = 26 [deprecated = true];</code>
-     * @param float $var
-     * @return $this
-     * @deprecated
-     */
-    public function setNsfwConfidence($var)
```

Comment:
Removing these is a breaking change. Do we know why this was done?

Reply:
@dwsupplee 
This happened in: `fix!: [Vision] removed fields from SafeSearchAnnotation message` (cl/446836546).  These fields been removed from the API for 3+ months. So this would be throwing API exceptions in some cases already. Even so, making this change would change the API error to a (less desirable) PHP Fatal error.

I see a few options:
1. Take the breaking change without a new major version. While it's not optimal, taking this change is better than not updating the API or creating a new major version. Also, these methods have been marked deprecated, so users have had time to upgrade.
2. Wait a longer period of time (for example, 6 months) from API deprecation before merging the change. These methods have been marked deprecated, so users have had time to upgrade.
3. Ignore the file in `owlbot.py`, or add the methods back using synthtool. This may have unintended consequences (segfaults on protobuf). 
4. Revert these changes from the PR manually every new Vision PR

I am okay with any of these, but the first is my vote.

Reply:
Thanks for the rundown of options @bshaffer - do we know definitively what behavior will occur when a user sets these fields? Does the upstream API throw an error or ignore the field, etc.?